core-library/src/main/java/org/silverpeas/core/contribution/publication/model/PublicationDetail.java

```diff
@@ -486,6 +489,11 @@ public PublicationDetail(PublicationPK pk, String name, String description, Date
 
   }
 
+  @Override
+  public PublicationI18N getTranslation(final String language) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1185)